src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -68,16 +68,20 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 	}
 
 	@HostListener('document:keydown', ['$event'])
-	onkeydown(e) {
-		// use preventDefault() to avoid invoking the editor's select all
-		// select the active .
-		if ((e.ctrlKey || e.metaKey) && e.key === 'a') {
-			e.preventDefault();
-			document.execCommand('selectAll');
-		}
-		if ((e.ctrlKey || e.metaKey) && e.key === 'z') {
-			e.preventDefault();
-			document.execCommand('undo');
+	onkeydown(e: KeyboardEvent) {
+		if (this.isActive()) {
+			// select the active .
+			if ((e.ctrlKey || e.metaKey) && e.key === 'a') {
```

Comment:
Has there ever been any discussion around these ignoring the keybinding shortcut settings?